packages/ui-date-time-input/src/DateTimeInput/__tests__/DateTimeInput.test.tsx

```diff
@@ -0,0 +1,688 @@
+        locale={locale}
+        timezone={timezone}
+        value={dateTime.toISOString()}
+        onChange={onChange}
+      />
+    )
+
+    const dateTimeInput = await DateTimeInputLocator.find()
+    expect(
+      await dateTimeInput.find(':contains(May 1, 2017 1:30 PM)')
+    ).to.exist()
+
+    await subject.setProps({ value: '2018-03-29T16:30Z' })
+
+    expect(
+      await dateTimeInput.find(':contains(March 29, 2018 12:30 PM)')
+    ).to.exist()
+  })
+
+  it('should update message when locale changed', async () => {
+    const onChange = stub()
+    const locale = 'en-US'
+    const timezone = 'US/Eastern'
+    const dateTime = DateTime.parse('2017-05-01T17:30Z', locale, timezone)
+
+    const subject = await mount(
+      <DateTimeInput
+        description="date time"
+        dateLabel="date"
+        prevMonthLabel="Previous month"
+        nextMonthLabel="Next month"
+        timeLabel="time"
+        invalidDateTimeMessage="whoops"
+        locale={locale}
+        timezone={timezone}
+        onChange={onChange}
+        value={dateTime.toISOString()}
+      />
+    )
+
+    const dateTimeInput = await DateTimeInputLocator.find()
+    expect(
+      await dateTimeInput.find(':contains(May 1, 2017 1:30 PM)')
+    ).to.exist()
+
+    await subject.setProps({
+      locale: 'fr'
+    })
+
+    expect(await dateTimeInput.find(':contains(1 mai 2017 13:30)')).to.exist()
+  })
+
+  it('should update message when timezone changed', async () => {
+    const onChange = stub()
+    const locale = 'en-US'
+    const timezone = 'US/Eastern'
+    const dateTime = DateTime.parse('2017-05-01T17:30Z', locale, timezone)
+
+    const subject = await mount(
+      <DateTimeInput
+        description="date time"
+        dateLabel="date"
+        prevMonthLabel="Previous month"
+        nextMonthLabel="Next month"
+        timeLabel="time"
+        invalidDateTimeMessage="whoops"
+        locale={locale}
+        timezone={timezone}
+        value={dateTime.toISOString()}
+        onChange={onChange}
+      />
+    )
+
+    const dateTimeInput = await DateTimeInputLocator.find()
+
+    expect(
+      await dateTimeInput.find(':contains(May 1, 2017 1:30 PM)')
+    ).to.exist()
+
+    await subject.setProps({
+      timezone: 'Europe/Paris'
+    })
+
+    expect(
+      await dateTimeInput.find(':contains(May 1, 2017 7:30 PM)')
+    ).to.exist()
+  })
+
+  it('should show error message if initial value is invalid', async () => {
+    await mount(
+      <DateTimeInput
+        description="date time"
+        dateLabel="date"
+        prevMonthLabel="Previous month"
+        nextMonthLabel="Next month"
+        timeLabel="time"
+        invalidDateTimeMessage="whoops"
+        value="totally not a date"
+      />
+    )
+    const dateTimeInput = await DateTimeInputLocator.find()
+    expect(await dateTimeInput.find(':contains(whoops)')).to.exist()
+  })
+
+  it('should show error message if initial defaultValue is invalid', async () => {
```

Comment:
Sorry, gave you a wrong example. Just adding `stub(console, 'error')` above the `await mount()` would be enough, because that removes all those nasty logs from the test.